core/math/color.h

```diff
@@ -174,16 +174,17 @@ struct _NO_DISCARD_ Color {
 
 	_FORCE_INLINE_ Color srgb_to_linear() const {
 		return Color(
-				r < 0.04045f ? r * (1.0f / 12.92f) : Math::pow((r + 0.055f) * (float)(1.0 / (1.0 + 0.055)), 2.4f),
```

Comment:
Multiply operations are slightly faster, so it cases like this, it's better to leave the multiplication in, otherwise you are adding CPU cycles where you don't have to. 

Further, There is a subtle but real difference between `* (1.0 / x)` and `/ x` which leads compilers to not optimize the line by replacing the divide with a multiply. 

So this PR is likely adding CPU cycles at run time. Of course all compilers are slightly different, so the impact is hard to predict, but it's best not to remove the multiply.